src/LogViewer/index.js

```diff
@@ -75,7 +75,7 @@ export default class LogViewer extends React.Component {
       parent.postMessage(JSON.stringify(qs), '*');
     }
 
-    history.pushState(null, '', `${location.origin}${location.pathname}?${stringify(qs)}`);
+    history.replaceState(null, '', `${location.origin}${location.pathname}?${stringify(qs)}`);
```

Comment:
`history.replaceState` fixes the problem of not going anywhere useful when clicking back. However, your history browsing history will still have all of the states listed. I'm not sure we can stop it from being recorded.